lib/awful/layout/init.lua

```diff
@@ -124,7 +124,7 @@ function layout.arrange(screen)
         local useless_gap = tag.getgap(tag.selected(screen),
                                        #client.tiled(screen))
         -- Handle padding
-        local padding = ascreen.padding(capi.screen[screen]) or {}
+        local padding = tag.getpadding(tag.selected(screen)) or ascreen.padding(capi.screen[screen]) or {}
```

Comment:
`ascreen.padding(capi.screen[screen])` is never used because `ascreen.padding(capi.screen[screen])` will return `tag.getproperty(t, "tag_padding") or beautiful.tag_padding or {}`